moveit_ros/moveit_servo/demos/cpp_interface/demo_pose.cpp

```diff
@@ -88,9 +88,9 @@ int main(int argc, char* argv[])
 
   // The dynamically updated target pose.
   PoseCommand target_pose;
-  target_pose.frame_id = servo_params.planning_frame;
+  target_pose.frame_id = "panda_link0";
```

Comment:
maybe constexpr base_frame and tip_frame at the top of the file?

Reply:
done